test/e2e/cypress/utils/sql_manager.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+Folder=$1
+Service=$2_db
+
+# Check we have enough arguments
+if [ $# -lt 3 ]
+then
+	echo "Missing expected arguments."
+	echo "sql_manager folder stack file1 file2 ..."
+	exit
+fi
+
+# Check the db service exists
+ID_Service=`docker ps | grep $Service | awk '{ print $1 }'`
+if [ -z "$ID_Service" ]
+then
+	echo ""
+	echo "couldn't find process matching '$Service' "
+	echo ""
+	echo "current processes :"
+	docker ps
+	echo ""
+	exit
+fi
+
+for FILE_SQL in "$@"; do
+	# Skip first 2 arguments
+	if [ $FILE_SQL == $1 ] || [ $FILE_SQL == $2 ]
+	then
+		continue
+	fi
+
+	DB_Init=`cat ./cypress/integration/$Folder/test_setup/sql/$FILE_SQL`
+	docker exec $ID_Service bash -c "echo '$DB_Init' > ./sql/$FILE_SQL"
+	docker exec $ID_Service bash -c "mysql -u root -proot \"appbuilder-admin\" < ./sql/$FILE_SQL"
```

Comment:
Are we making the assumption that the tests are always run against `appbuilder-admin` tenant?

Reply:
I think for most of our tests should run on the admin tenant. We will need to add tests to test tenant features, so it might be something to look at in the future.